options/glibc/include/sys/kd.h

```diff
@@ -0,0 +1,17 @@
+#ifndef _SYS_KD_H
+#define _SYS_KD_H
+
+/* Make sure the <linux/types.h> header is not loaded.  */
```

Comment:
Is this file copied from musl? The spacing should also be fixed to match mlibc's style.

Reply:
Based on indeed, I'll fix the spacing.

Reply:
What is this actually used for?

Reply:
As far as I'm aware, this is just some glibc compatibility header.

Reply:
I meant the "Make sure linux/types.h is not loaded". What is that used for? Our ``linux/kd.h`` does not have any inclusion of ``linux/types.h``.